openreview/conference/templates/programchairWebfield.js

```diff
@@ -196,6 +210,9 @@ var getUserProfiles = function(userIds) {
 };
 
 var findProfile = function(profiles, id) {
+  // if (id.hasOwnProperty('id')){
```

Comment:
I would either remove this commented code or add a comment explaining why it has to remain here